src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -0,0 +1,142 @@
+		// subtitle exists
+		XCTAssertTrue(app.staticTexts[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.descriptionSubHeadline].exists)
+		// legal text exists
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.acknowledgementTitle].exists)
+		
+		let dataPrivacyButton = try XCTUnwrap(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.dataPrivacyTitle])
+		dataPrivacyButton.tap()
+		
+		/// Legal Text Screen
+		
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.dataPrivacyTitle].exists)
+		
+		let backButton = try XCTUnwrap(app.navigationBars.buttons.element(boundBy: 0))
+		backButton.tap()
+		
+		/// -> Antigen Test Information Screen
+		
+		let continueButton = app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Information.continueButton]
+		XCTAssertTrue(continueButton.waitForExistence(timeout: .short))
+		continueButton.tap()
+		
+		/// Create Antigen Test Profile Screen
+
+		let saveProfileButton = try XCTUnwrap(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Create.saveButton])
+		XCTAssertFalse(saveProfileButton.isEnabled)
+		
+		let firstNameTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.firstNameTextField])
+		firstNameTextField.tap()
+		firstNameTextField.typeText("Bastian")
+		
+		let lastNameTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.lastNameTextField])
+		lastNameTextField.tap()
+		lastNameTextField.typeText("Kohlbauer")
+		
+		let birthDateTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.birthDateTextField])
+		birthDateTextField.tap()
+		birthDateTextField.typeText("15-12-1986")
+		
+		let streetTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.streetTextField])
+		streetTextField.tap()
+		streetTextField.typeText("Herr Bastian Kohlbauer Str. 1")
+		
+		let postalCodeTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.postalCodeTextField])
+		postalCodeTextField.tap()
+		postalCodeTextField.typeText("80639")
+		
+		let cityTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.cityTextField])
+		cityTextField.tap()
+		cityTextField.typeText("München")
+		
+		let phoneNumberTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.phoneNumberTextField])
+		phoneNumberTextField.tap()
+		phoneNumberTextField.typeText("089123456")
+		
+		let emailAddressTextField = try XCTUnwrap(app.cells.textFields[AccessibilityIdentifiers.AntigenProfile.Create.emailAddressTextField])
+		emailAddressTextField.tap()
+		emailAddressTextField.typeText("bastian@bastian.codes")
```

Comment:
for every tab, you should check with .exists or .waitForExistence()

Reply:
Why do we need this?